core/build.gradle

```diff
@@ -76,6 +76,7 @@ test {
 
     // Working dir will be root project
     workingDir = rootDir
+    testLogging.showStandardStreams = true
```

Comment:
Well, can comment it out.

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 12:17 AM, Jakub Háva <notifications@github.com>
wrote:

> In core/build.gradle
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76191849>:
>
> > @@ -76,6 +76,7 @@ test {
> >
> >      // Working dir will be root project
> >      workingDir = rootDir
> > +    testLogging.showStandardStreams = true
>
> Do we need to show console output during running tests ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76191849>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuKyprTAFfXTg1NWYORhUwTOEw-zeRks5qjUGZgaJpZM4Jsgi_>
> .
>